zulipterminal/ui_tools/views.py

```diff
@@ -729,7 +730,12 @@ def users_view(self, users: Any = None) -> Any:
             unread_count = self.view.model.unread_counts["unread_pms"].get(
                 user["user_id"], 0
             )
+            realm_enable_guest_user_indicator = self.model.initial_data[
+                "realm_enable_guest_user_indicator"
+            ]
```

Comment:
Hey for this ill use 
realm_enable_guest_user_indicator = self.model.initial_data.get(
                "realm_enable_guest_user_indicator", None
            )
so that it returns None when not present, also are you suggesting not to use this here ?

Reply:
Right, that will avoid the exception.

As per another comment, in (new) code we are aiming to have server changes (different versions) isolated in the model, so that UI code isn't dependent on that.